graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/SchemaGenerator.kt

```diff
@@ -17,101 +17,37 @@
 package com.expediagroup.graphql.generator
 
 import com.expediagroup.graphql.SchemaGeneratorConfig
-import com.expediagroup.graphql.TopLevelObject
-import com.expediagroup.graphql.generator.extensions.getKClass
-import com.expediagroup.graphql.generator.extensions.isEnum
-import com.expediagroup.graphql.generator.extensions.isInterface
-import com.expediagroup.graphql.generator.extensions.isListType
-import com.expediagroup.graphql.generator.extensions.isUnion
-import com.expediagroup.graphql.generator.extensions.wrapInNonNull
 import com.expediagroup.graphql.generator.state.SchemaGeneratorState
-import com.expediagroup.graphql.generator.state.TypesCacheKey
-import com.expediagroup.graphql.generator.types.generateEnum
-import com.expediagroup.graphql.generator.types.generateInputObject
-import com.expediagroup.graphql.generator.types.generateInterface
-import com.expediagroup.graphql.generator.types.generateList
-import com.expediagroup.graphql.generator.types.generateMutations
-import com.expediagroup.graphql.generator.types.generateObject
-import com.expediagroup.graphql.generator.types.generateQueries
-import com.expediagroup.graphql.generator.types.generateScalar
-import com.expediagroup.graphql.generator.types.generateSubscriptions
-import com.expediagroup.graphql.generator.types.generateUnion
 import graphql.schema.GraphQLCodeRegistry
-import graphql.schema.GraphQLSchema
 import graphql.schema.GraphQLType
-import graphql.schema.GraphQLTypeReference
-import graphql.schema.GraphQLTypeUtil
-import kotlin.reflect.KClass
-import kotlin.reflect.KType
 
 open class SchemaGenerator(val config: SchemaGeneratorConfig) {
```

Comment:
this change make the class pretty useless -> it is called schema generator yet it doesn't generate schema at all

Reply:
Yea it is now just a schema generator state. But that is already taken. We could combine  the state and generator into one class?

Reply:
I have deleted SchemaGeneratorState